migrations/20210208130000-add-2fa-recovery-codes-to-user.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'twoFactorAuthRecoveryCodes', {
+      type: Sequelize.ARRAY(Sequelize.STRING),
+      allowNull: true,
+      defaultValue: null,
+    });
+    await queryInterface.addColumn('UserHistories', 'twoFactorAuthRecoveryCodes', {
+      type: Sequelize.ARRAY(Sequelize.STRING),
+      allowNull: true,
+      defaultValue: null,
+    });
+  },
```

Comment:
Lets's look at it closely when this is deployed to production, @Betree or myself should be aware there is a migration going on.